node/src/components/linear_chain_sync/event.rs

```diff
@@ -0,0 +1,85 @@
+use crate::types::{ActivationPoint, Block, BlockHash, NodeId};
```

Comment:
I think the entire linear_chain_sync component has been removed on the feat-fast-sync branch.